tests/gh-api-helpers.js

```diff
@@ -0,0 +1,210 @@
+const { Octokit } = require('octokit')
+const axios = require('axios')
+const AdmZip = require('adm-zip')
+
+const ref = process.env.GITHUB_HEAD_REF || process.env.GITHUB_REF
+const owner = 'Kong'
+const repo = 'gateway-test-scheduler'
+const workflow_id = 'integration.yml'
+
+const POLLING_TIMEOUT = 300000
+const RETRY_INTERVAL = 5000
+
+let octokit
```

Comment:
Is there a particular reason why you can't initialize `octokit` here and make it a `const`ant?  That'd also remove the need for the undefined checks below.

Reply:
no reason, I'm changing it.